targets/LuaSdk/make.js

```diff
@@ -143,7 +143,7 @@ function getVerticalNameDefault() {
 function getRequestActions(tabbing, apiCall) {
     var requestAction = "";
 
-    if (apiCall.url === "/Authentication/GetEntityToken")
+    if (apiCall.url === "/Authentication/GetEntityToken" || apiCall.url === "/GameServerIdentity/AuthenticateGameServerWithCustomId")
```

Comment:
The handling of the AuthenticateGameServerWithCustomId request needs to look a little different from GetEntityToken. The only auth header that is valid is X-EntityToken. The others won't work so I don't think it needs to be included in this "if" statement. The X-EntityToken header should be automatically attached as it is for other Entity APIs

Reply:
This comment applies across multiple other SDKs as well.